gobblin-data-management/src/main/java/org/apache/gobblin/data/management/copy/CopyConfiguration.java

```diff
@@ -65,6 +65,10 @@
 
   public static final String ABORT_ON_SINGLE_DATASET_FAILURE = COPY_PREFIX + ".abortOnSingleDatasetFailure";
 
+  /*
+   * Config to store different classes of rejected requests. Possible values are "all","none", or "min" (default).
```

Comment:
This should be enum just like defined in ConcurrentBoundedPriorityIterable.java
Please consider creating an enum class for STORE_REJECTED_REQUESTS to avoid duplication.

Reply:
I moved the Enum to RequestAllocatorConfig. Both CopyConfiguration and  ConcurrentBoundedPriorityIterable classes can access the Enum class without duplication.